vite.config.js

```diff
@@ -10,7 +10,7 @@ export default defineConfig(({ command, mode }) => {
 
   return {
     base: command === 'serve' ? '' : '/dist/',
-    publicDir: './web/dist',
+    publicDir: './web',
```

Comment:
@brianhanson File resolution in stylesheets failed wtih your original value. After this change file resolution seems to work and didn't seem to affect either the dev or production build at all.